pkgs/top-level/aliases.nix

```diff
@@ -69,6 +69,8 @@ mapAliases ({
   adtool = throw "'adtool' has been removed, as it was broken and unmaintained";
   adom = throw "'adom' has been removed, as it was broken and unmaintained"; # added 2024-05-09
   advcpmv = throw "'advcpmv' has been removed, as it is not being actively maintained and break recent coreutils."; # Added 2024-03-29
+  # Post 24.11 branch-off, this should throw an error
+  addOpenGLRunpath = addDriverRunpath; # Added 2024-05-25
```

Comment:
I think it's OK to make this a `lib.warn` even

Reply:
I tried that in the first iteration of this PR, but ofborg will complain as it'll try to evaluate `addOpenGLRunpath`

Reply:
If we could ensure it doesn't complain later in other people's PRs... regardless, let's just merge this as is

Reply:
Actually. I'm pretty sure `warn`s in `aliases.nix` are allowed. Ofborg should only complain if there's a derivation left that actually asks for `addOpenGLRunpath` instead of `addDriverRunpath`